var/spack/repos/builtin/packages/numactl/package.py

```diff
@@ -25,8 +28,9 @@ class Numactl(AutotoolsPackage):
     patch("numactl-2.0.11-sysmacros.patch", when="@2.0.11")
     # https://github.com/numactl/numactl/issues/94
     patch("numactl-2.0.14-symver.patch", when="@2.0.14")
-    patch("fix-empty-block.patch", when="@2.0.10:2.0.14")
+    patch("fix-empty-block.patch", when="@2.0.10:2.0.16")
     patch("link-with-latomic-if-needed.patch", when="@2.0.14")
```

Comment:
Based on https://github.com/spack/spack/pull/46150/files#r1742322060 I think this is the patch sequence then?
```suggestion
    patch("link-with-latomic-if-needed.patch", when="@2.0.14")
    patch("link-with-latomic-if-needed-v2.0.16.patch", when="@2.0.16")
```
Skipping 2.0.15 here too per summary.

Reply:
Oops. Yep. I forgot to add that patch to `package.py`. Thanks!